### ISSUE TITLE
Epic Login URL Changed?

### ISSUE THREAD

**Issue by networldsystems** [JosefNemec/Playnite#1483]
I tried to update Epic games in Playnite and received a "user not authenticated" notification.

Went to Settings > Libraries > Epic and clicked on the Authenticate button. I get a 404 from Epic's site:
<img width="482" alt="Epic_404" src="https://user-images.githubusercontent.com/26266261/66274143-16876100-e841-11e9-8903-0e52716bad07.png">

**Comment by JosefNemec** [comment]
I just tried it and it works for me, I'm not sure why it's not working for you.

![image](https://user-images.githubusercontent.com/3874087/66274314-929f9a00-e87d-11e9-89a6-8a904f98cf20.png)

**Comment by networldsystems** [comment]
Thanks for checking. I located the first exception encountered in the Playnite log:2019-10-06 13:31:50.8042|DEBUG:Found 18 installed Epic games.
2019-10-06 13:31:51.4272|ERROR:Failed to import uninstalled Epic games.System.FormatException: The format of value ' ' is invalid.
   at System.Net.Http.Headers.HttpHeaderParser.ParseValue(String value, Object storeValue, Int32& index)
   at System.Net.Http.Headers.HttpHeaders.ParseAndAddValue(String name, HeaderStoreItemInfo info, String value)
   at System.Net.Http.Headers.HttpHeaders.Add(String name, String value)
   at EpicLibrary.Services.EpicAccountClient.<invokeRequest>d__15`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at EpicLibrary.Services.EpicAccountClient.GetIsUserLoggedIn()
   at EpicLibrary.Services.EpicAccountClient.GetAssets()
   at EpicLibrary.EpicLibrary.GetLibraryGames()
   at EpicLibrary.EpicLibrary.GetGames()
2019-10-06 13:32:11.6141|ERROR:Failed to validation Epic authentication.System.FormatException: The format of value ' ' is invalid.
   at System.Net.Http.Headers.HttpHeaderParser.ParseValue(String value, Object storeValue, Int32& index)
   at System.Net.Http.Headers.HttpHeaders.ParseAndAddValue(String name, HeaderStoreItemInfo info, String value)
   at System.Net.Http.Headers.HttpHeaders.Add(String name, String value)
   at EpicLibrary.Services.EpicAccountClient.<invokeRequest>d__15`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at EpicLibrary.Services.EpicAccountClient.GetIsUserLoggedIn()

Subsequent exceptions for additional attempts at Epic authenticating look the same as above.
I'll have to try it again after a reboot.
Thanks,
Steve

    On Sunday, October 6, 2019, 2:09:44 PM CDT, Josef Nemec <notifications@github.com> wrote:  
 
 
I just tried it and it works for me, I'm not sure why it's not working for you.

**Comment by JosefNemec** [comment]
So is this still an issue?

**Comment by networldsystems** [comment]
No sir. Before I originally reported it, I had checked that my version was up-to-date. It reported it was current, but was not (could have been a user issue on my part). When I checked manually, it was 5.5 and the current version was 5.8. I updated to 5.8 and the issue was resolved. My apologies for not following up.
Thanks,
Steve

    On Monday, October 21, 2019, 7:08:43 AM CDT, Josef Nemec <notifications@github.com> wrote:  
 
 
So is this still an issue?
